src/main.js

```diff
@@ -137,19 +138,19 @@ export default class Auth {
 	}
 
 	/**
-	 * Makes sure the user is signed-in and has up-to-date credentials.
-	 * @throws Will throw if the user is not signed-in.
+	 * Makes sure the user is signed in and has up-to-date credentials.
+	 * @throws Will throw if the user is not signed in.
 	 * @private
 	 */
 	async enforceAuth() {
-		if (!this.user) throw Error('The user must be signed-in to use this method.');
+		if (!this.user) throw Error('The user must be signed in to use this method.');
 		return this.refreshIdToken(); // Won't do anything if the token is valid.
 	}
 
 	/**
 	 * Updates the user data in localStorage.
 	 * @param {Object} userData New user data.
-	 * @param {boolean} [updateStorage = true] Check whether to update localStorage or not.
+	 * @param {boolean} [persist = true] Whether to update local storage or not.
```

Comment:
should be `localStorage` for consistency with the rest of comments

Reply:
true, I'll fix it and another one in line 260.